src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const paymentPerDay = 40;
+  const discount = 20;
+  const bigDiscount = 50;
+
+  if (days >= 3 && days < 7) {
+    return days * paymentPerDay - discount;
+  }
+
+  if (days >= 7) {
+    return days * paymentPerDay - bigDiscount;
+  }
```

Comment:
Swap the first and the second checks and remove && < 7.
In that case, the logic is next: 
1. If days >=7 - return sum with big discount
2. if days >=3 - return sum ith regular discount
3. if not >=7 or not >=3 - return just sum

Your solution is correct but always try to simplify code